Assets/_BForBoss/_Core/Scripts/Boss/DerekBossManager.cs

```diff
@@ -121,6 +128,8 @@ public void OnCollided(Vector3 collisionPoint, Vector3 collisionNormal)
                 return;
             }
 
+            _currentHealth--;
+            _derekHealthView.SetState(new DerekHealthViewState((float)_currentHealth / _healthAmount, _vulnerability == DerekContextManager.Vulnerability.Invulnerable));
             _onVulnerabilityExpired?.Invoke(true);
```

Comment:
@Gursaanj It seems like this is important? but i don't know how this would effect derek